meetings/2023-06-07.md

```diff
@@ -72,21 +72,24 @@ URL: https://www.w3.org/2023/09/TPAC/
 ### HTTPSig Auth Demo
 
 * LD: What' the status of HttpSig specification?
-* HS: HTTP Signatures in IETF. Last call was in September. HTTPSig is a lot more Solid now.
-* HS: Written in Scala that compiles to JavaScript.
+* HS: [HTTP Signatures](https://datatracker.ietf.org/doc/draft-ietf-httpbis-message-signatures/) in being developed in the IETF HTTPBis group. The authors were in last-call in September at version 13. The received enough feedback to get them to version 17 in the meantime. But it is close to finished. HTTPSig is a lot more Solid now.
+* HS: [My implementation of httpSig is on github](https://github.com/bblfish/httpSig) , It is written in Scala that compiles to JavaScript, so that I can be used on the client and on the server. (It would need work for nodeJS) 
```

Comment:
```suggestion
* HS: [My implementation of httpSig is on github](https://github.com/bblfish/httpSig). It is written in Scala that compiles to JavaScript, so that it can be used on the client and on the server. (It would need work for nodeJS.) 
```